lib/PrimitiveHelpers.js

```diff
@@ -163,6 +186,14 @@ function transformPrimitives(gltf, primitives, transform) {
     }
 }
 
+/**
+ * Get a hash table like object with the material as the hash and the
+ * primitives that use the material in an array as the bucket.
+ *
+ * @param {Array} primitives An array containing the primitive to sort.
+ * @returns {Object} An object with properties names of material types where
+ * each property is an array conmtaining all the primitives that use that material.
```

Comment:
Put the doc on one line, which may be required by jsdoc.

Description can be more concise: "Returns an object mapping material ids to an array of primitives"